packages/@aws-cdk/aws-apigateway/lib/method.ts

```diff
@@ -192,6 +192,9 @@ export class Method extends Resource {
       authorizer._attachToApi(this.api);
     }
 
+    const integration = props.integration ?? this.resource.defaultIntegration ?? new MockIntegration();
```

Comment:
Is this another bug that's being fixed here?

Reply:
No, this is code being moved from `renderIntegration()` method below.

See the first few lines of this method below that is being deleted.

Reply:
ok cool, missed that